lib/papertrail/cli.rb

```diff
@@ -137,12 +142,31 @@ def query_time_range
       end
     end
 
+    COLORS = [:cyan, :yellow, :green, :magenta, :red]
+
+    def colorize(event)
+      idx = (event.data["hostname"].length + event.data["program"].length) % 5
```

Comment:
This is a great idea. I see one problem using lengths, though. If I have the hosts `web1`, `web2`, etc, the same program on all of them will be the same color.

<img width="289" alt="screen shot 2015-11-01 at 10 49 10 am" src="https://cloud.githubusercontent.com/assets/23582/10869749/6adc9fd0-8086-11e5-8416-7ce9160e8411.png">

What if we just summed up the unicode value of each character?

```ruby
idx = (event.data["hostname"] + event.data["program"]).unpack('U*').inject(&:+) % 5
```

<img width="290" alt="screen shot 2015-11-01 at 10 49 40 am" src="https://cloud.githubusercontent.com/assets/23582/10869751/709ed712-8086-11e5-810e-9c7cd2e768ba.png">

I don't know if this is the best way to get an integer value from a given string. It's just the first thing that came to mind.